# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xfce4-mixer/devfs.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./lib/vc_oss.c.orig	2004-12-28 00:05:07.000000000 -0300
+++ ./lib/vc_oss.c	2004-12-28 00:05:40.000000000 -0300
@@ -72,7 +72,7 @@
 
 #define MAX_AMP 100
 
-static char dev_name[PATH_MAX] = { "/dev/mixer" };
+static char dev_name[PATH_MAX] = { "/dev/sound/mixer" };
 
 static int mixer_handle = -1;
 static int devmask = 0;                /* Bitmask for supported mixer devices */
--- ./panel-plugin/test_vc.c.orig	2004-12-28 00:04:43.000000000 -0300
+++ ./panel-plugin/test_vc.c	2004-12-28 00:05:51.000000000 -0300
@@ -31,7 +31,7 @@
 	vc_get_volume (NULL);
 	vc_get_volume (NULL);*/
 	
-	vc_set_device ("/dev/mixer1");
+	vc_set_device ("/dev/sound/mixer1");
 
 /*	vc_get_volume (NULL);
 	vc_get_volume (NULL);
